app/src/main/java/com/example/criminalintent/presentation/crime_detail/CrimeDetailViewModel.kt

```diff
@@ -24,6 +24,11 @@ class CrimeDetailViewModel : ViewModel() {
             crimeRepository.getCrime(crimeId)
         }
 
+    fun updateSuspect(suspect: String) {
```

Comment:
This should be a response to an action. The action here is that the suspect's name has been selected.
The parameter is the suspect's name, not the suspect model